amaranth-chrome-ext/src/CalorieLabeller.js

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
Nit: This might be able to be a single line.

Reply:
Fixed in https://github.com/googleinterns/amaranth/pull/44/commits/a0a1070e1663d24d48d40ddb484135cc32529d60